tests/Dialect/BGV/Conversions/bgv_to_openfhe/bgv_to_openfhe.mlir

```diff
@@ -1,33 +1,47 @@
 // RUN: heir-opt --mlir-print-local-scope --bgv-to-openfhe %s | FileCheck %s
 
-#encoding = #lwe.polynomial_evaluation_encoding<cleartext_start=30, cleartext_bitwidth=3>
+!Z1032955396097_i64_ = !mod_arith.int<1032955396097 : i64>
+!Z1095233372161_i64_ = !mod_arith.int<1095233372161 : i64>
+!Z65537_i64_ = !mod_arith.int<65537 : i64>
 
-#my_poly = #polynomial.int_polynomial<1 + x**1024>
-// cmod is 64153 * 2521
-#ring1 = #polynomial.ring<coefficientType=!mod_arith.int<161729713:i32>, polynomialModulus=#my_poly>
-#ring2 = #polynomial.ring<coefficientType=!mod_arith.int<2521:i32>, polynomialModulus=#my_poly>
+!rns_L0_ = !rns.rns<!Z1095233372161_i64_>
+!rns_L1_ = !rns.rns<!Z1095233372161_i64_, !Z1032955396097_i64_>
 
-#params1 = #lwe.rlwe_params<dimension=2, ring=#ring1>
-#params2 = #lwe.rlwe_params<dimension=4, ring=#ring1>
-#params3 = #lwe.rlwe_params<dimension=2, ring=#ring2>
-#params4 = #lwe.rlwe_params<dimension=3, ring=#ring1>
+#ring_Z65537_i64_1_x1024_ = #polynomial.ring<coefficientType = !Z65537_i64_, polynomialModulus = <1 + x**1024>>
+#ring_rns_L0_1_x1024_ = #polynomial.ring<coefficientType = !rns_L0_, polynomialModulus = <1 + x**1024>>
+#ring_rns_L1_1_x1024_ = #polynomial.ring<coefficientType = !rns_L1_, polynomialModulus = <1 + x**1024>>
```

Comment:
File a followup issue for verifier of modulus chain level with actual underlying Polynomial coefficient type